src/ui/server/api/external/index.ts

```diff
@@ -1,19 +1,37 @@
 import dotenv from 'dotenv';
 import axios, { AxiosResponse, RawAxiosRequestConfig } from 'axios';
+import { EXTERNAL_API_ENDPOINTS } from '../../constants/external-apis';
 
 dotenv.config();
 
+const { APIM_MDM } = EXTERNAL_API_ENDPOINTS;
 const headers = {
   'Content-Type': 'application/json',
   [String(process.env.APIM_MDM_KEY)]: process.env.APIM_MDM_VALUE,
 };
 
 const external = {
+  getCountries: async () => {
```

Comment:
This is not valid, this has been replaced with a UI call to our own API, which then calls APIM CIS endpoint. Do you have the latest main-application`?

Reply:
Have removed.